draft-js-mention-plugin/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Fixed race condition where the SuggestionPortal would unregister and not register again when inputting Japanese, etc.
 - Fixed bug where `mentionPrefix` does not appear in `editorState`. `mentionPrefix` is no longer passed to `mentionComponent`.
 - Fixed bug where `onSearchChange` didn't fire when a user switched between two different mention autocompletions with the same search value. Now it will trigger `onSearchChange` in such a case.
+- Fixed unrecognized `isFocused` React prop.
```

Comment:
this would need to be added under a new version. Please add a new version to the changelog and bump the package version